airbyte-integrations/connectors/destination-clickhouse/src/main/java/io/airbyte/integrations/destination/clickhouse/ClickhouseSqlOperations.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.clickhouse;
+
+import io.airbyte.db.jdbc.JdbcDatabase;
+import io.airbyte.integrations.base.JavaBaseConstants;
+import io.airbyte.integrations.destination.jdbc.JdbcSqlOperations;
+import io.airbyte.protocol.models.AirbyteRecordMessage;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.sql.SQLException;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.yandex.clickhouse.ClickHouseConnection;
+import ru.yandex.clickhouse.ClickHouseStatement;
+import ru.yandex.clickhouse.domain.ClickHouseFormat;
+
+public class ClickhouseSqlOperations extends JdbcSqlOperations {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(ClickhouseSqlOperations.class);
+
+  @Override
+  public void createSchemaIfNotExists(final JdbcDatabase database, final String schemaName) throws Exception {
+    database.execute(String.format("CREATE DATABASE IF NOT EXISTS %s;\n", schemaName));
+  }
+
+  @Override
+  public String createTableQuery(final JdbcDatabase database, final String schemaName, final String tableName) {
+    return String.format(
+        "CREATE TABLE IF NOT EXISTS %s.%s ( \n"
+            + "%s String,\n"
+            + "%s String,\n"
+            + "%s DateTime64(3, 'GMT') DEFAULT now(),\n"
+            + "PRIMARY KEY(%s)\n"
+            + ")\n"
+            + "ENGINE = MergeTree;\n",
```

Comment:
ClickHouse has different table Engines to use.
Maybe it make sens to point on UI that we are going to use this table engine type.

There are family of MergeTree engines in ClickHouse - https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/

Also kind of simple Log table Engines
https://clickhouse.com/docs/en/engines/table-engines/log-family/

@sherifnada should we consider to use different table engines for ClickHouse?
Thanks

Reply:
That `createTableQuery` is for raw airbyte table, it is mainly for staging purpose so I think end-user may not care about its underlying engine, but I am okay to change it if you have better idea.